README.md

```diff
@@ -118,10 +117,11 @@ karmaConfig: {
 *Note*: Whichever testFramework you use should also be reflected in the `testFramework` property of stryker itself. For example: `testFramework: 'mocha'`  
 
 Not all karma config can be overriden, as Stryker requires specific functionality from the testRunner to do its magic. 
-Karma config that *cannot* be overriden:
 
-* `files`: The karma-runner will fill this based on the `files` and `filesToMutate` configuration in the `stryker-conf.js` file.
-* `coverageReporter`: This will be enabled for the initial test run, disabled for testing the mutants.
+The following Karma config options cannot be overridden:
+
+* `files`: The karma-runner will fill this based on the `files` and `mutate` configuration in the `stryker-conf.js` file (or your `karma.conf.js` file when using the *automatic setup*).
+* `coverageReporter`: Stryker-karma-runner will use its own coverage reporter for the initial test run and disabled it for testing the mutants.

```

Comment:
`coverageReporter`: For the initial test run, the `stryker-karma-runner` will use its own coverage reporter. For testing the mutants, however, it will be disabled.